frontend/src/PrivateRoute.tsx

```diff
@@ -5,9 +5,14 @@ import useKeycloakWrapper from 'hooks/useKeycloakWrapper';
 interface IPrivateRouteProps extends RouteProps {
   // tslint:disable-next-line:no-any
   component: any;
-  role?: string;
+  role?: string | Array<string>;
+  claim?: string | Array<string>;
```

Comment:
I'd rather have this property be renamed to `claims` and always make it an array. It's cleaner and supports the single claim via `claims = [ 'single-claim' ]`

Reply:
Not sure I agree entirely.  The way it is allows for overriding.  No different than what we'd do in languages that support overriding natively.

Reply:
not really, by having the code support string or array of strings you will have to check whether it is a string or an array. I don't see the need for the added complexity

Reply:
The more common use is a single role/claim.  The code has already been update to check.